0015-ilp-addresses/0015-ilp-addresses.md

```diff
@@ -0,0 +1,123 @@
+# ILP Addresses
+
+_ILP addresses_ provide a way route payments to their intended destination through a recursive series of hops, including any number of ILP Connectors. (This happens after the payment is set up on a higher level, such as [SPSP](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md).) Addresses can be subdivided into two categories:
+
+- **Destination Addresses** are complete addresses that can receive payments. A destination address always maps to one account in a ledger. (It can also provide more specific information, such as an invoice ID or a sub-account.) Destination addresses MUST NOT end in a period (`.`) character.
+- **Prefix Addresses** are incomplete addresses representing a group of destination addresses. Many depths of grouping are possible: groups of accounts or sub-accounts, an individual ledger or subledger, or entire neighborhoods of ledgers. Prefix addresses MUST end in a period (`.`) character. Payment setup protocols MUST reject payments to prefix addresses.
+
+Both types of address usually contain one or more period characters as separators.
+
+## Address Requirements
+
+ILP Addresses must meet the following requirements:
+
+1. The address MUST begin with a prefix indicating the allocation scheme. The only supported allocation scheme for now is the [General Allocation Scheme](#general-allocation-scheme), which uses the prefix `g.`.
```

Comment:
Connectors in the same "neighborhoods" are close to one another in the sense that they are likely to be able to route to one another. Connectors can explicitly say they can route to addresses with a different allocation scheme (by broadcasting routes that don't start with `g.`) but I don't think it makes sense to mix schemes.